### PR TITLE
Add `RiskCorrelationID` to `BTPayPalVaultEditRequest`

### PR DESCRIPTION
**Pull request by KunJeongPark**
### Summary of changes

- Add `RiskCorrelationID` to `BTPayPalVaultEditRequest`
- Add docStrings for `RiskCorrelationID`

### Checklist

- [ ] Added a changelog entry

- @KunJeongPark @jwarmkessel

### CODE REVIEW

**Review by stechiu — APPROVED** [review]

File: Sources/BraintreePayPal/BTPayPalVaultEditRequest.swift [hunk 1]
```diff
@@ -6,15 +6,18 @@ public struct BTPayPalVaultEditRequest {
...
     private let editPayPalVaultID: String
     private let merchantAccountID: String?
+    private let riskCorrelationID: String?
 
     //   TODO: specify endpoint for merchant to retrieve the token
     /// Initializes a PayPal Edit Request for the edit funding instrument flow
     /// - Parameters:
     ///   - editPayPalVaultID: Required: The `edit_paypal_vault_id` returned from the server side request
     ///   merchantAccountID: optional ID of the merchant account; if one is not provided the default will be used
+    ///   riskCorrelationID: optional ID passed in by merchant to track errors
```
Comment (jaxdesmarais) [inline Sources/BraintreePayPal/BTPayPalVaultEditRequest.swift:16]:
```suggestion
    ///   merchantAccountID: Optional: ID of the merchant account; if one is not provided the default will be used
    ///   riskCorrelationID: Optional: Required only for the error handling flow to retry failed attempts. 
```

Comment (KunJeongPark) [reply Sources/BraintreePayPal/BTPayPalVaultEditRequest.swift:16]: Thanks Jax, I will include this as we get updated req's on this.

Comment (jaxdesmarais) [reply Sources/BraintreePayPal/BTPayPalVaultEditRequest.swift:16]: Lets update to the suggestion above and we should be all set for this PR.

Comment (KunJeongPark) [reply Sources/BraintreePayPal/BTPayPalVaultEditRequest.swift:16]: I am closing this PR with updated req's.

**Pull request closed without merging**